src/qml/WelcomeScreen.qml

```diff
@@ -441,6 +444,22 @@ Page {
                 return Math.min( result + padding * 2,mainWindow.width - 20);
             }
 
+            MenuItem {
+              id: fileAssociationProject
+              visible: recentProjectActions.recentProjectType != 2;
+
+              font: Theme.defaultFont
+              width: parent.width
+              height: visible ? 48: 0
+              leftPadding: 10
```

Comment:
```suggestion
              leftPadding: 10
              checkable: true
              checked: settings.value( '/QField/fileAssociationProject', false )
```

Just an idea ?

Reply:
@m-kuhn , if we use checked states, do we then not change the item text's set<->unset verb?

Reply:
We do that, would mean to change it to "Base map project for file association" (and I could also see another one "Auto open project" which auto opens the project on QField launch)

Reply:
OK, updating.